contracts/MintShop1155.sol

```diff
@@ -492,21 +363,14 @@ contract MintShop1155 is Sweepable, ReentrancyGuard {
   /**
     Construct a new shop which can mint items upon purchase from various pools.
 
-    @param _owner The address of the administrator governing this collection.
     @param _paymentReceiver The address where shop earnings are sent.
     @param _globalPurchaseLimit A global limit on the number of items that a
       single address may purchase across all item pools in the shop.
   */
-  constructor(address _owner, address _paymentReceiver,
```

Comment:
Please keep `_owner` with the optional ownership transfer in the constructor; we like using this pattern to support deploying a contract on behalf of someone else. Unless you think there's a good reason why it should be removed.